packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -175,7 +175,7 @@ export const getStyles = (props: IMessageBarStyleProps): IMessageBarStyles => {
               [HighContrastSelector]: {
                 background: 'WindowText',
                 color: 'Window',
-                content: ' '
+                content: '""'
```

Comment:
er, why do we even have this &:before here??

Reply:
I honestly have no idea, I just noticed it was not being applied at all due to the improper `content` style. I will check it in high contrast mode and get back real quick to see if it's needed.

Reply:
Ok. I removed the `:before` completely because it looks like me fixing the `content` style actually broke the intention behind it. It looks like it was not needed in here at all.